src/nester/templates/c/c_layout.json

```diff
@@ -1,14 +1,14 @@
 {
   "$projectname": {
     "include": {
-      "my_header.h": {}
+      "my_header.h": "#ifndef MY_HEADER_H\n#define MY_HEADER_H\n\nvoid my_function();\n\n#endif /* MY_HEADER_H */"
```

Comment:
IMO, `#pragma once` is nicer than this style of include guard, though it's not oficially part of the C and C++ specifications

Reply:
Interesting never heard of it. However, I think it would be best to stick to the offical specifications in this case.